xml/FrameworksIndex/netframework-4.7.2.xml

```diff
@@ -15266,6 +15266,7 @@
     </Type>
     <Type Name="System.Lazy`1" Id="T:System.Lazy`1">
       <Member Id="M:System.Lazy`1.#ctor" />
+      <Member Id="M:System.Lazy`1.#ctor(`0)" />
```

Comment:
I don't understand where this new member is coming from @dend @joelmartinez

We only had 6 ctors on MSDN and this adds a 7th. I don't see this as well on ildasm?

![image](https://user-images.githubusercontent.com/12971179/45856186-59df6200-bd08-11e8-861f-a859d07fdb2c.png)

Reply:
Ok ... so that particular ctor can be found in netcore 2 and 2.1. So it's a valid ctor, however, I'm not sure how it came to be referenced in that framework index for 4.7. I'm going to look into that now

Reply:
Thanks @joelmartinez. I'll merge this and then open a bug.